src/utils/permalinks/Permalinks.ts

```diff
@@ -241,22 +238,25 @@ export class RoomPermalinkCreator {
     }
 
     private updateServerCandidates = () => {
-        let candidates = [];
+        const candidates = [];
         if (this.highestPlUserId) {
             candidates.push(getServerName(this.highestPlUserId));
```

Comment:
Should we maybe use a Set here so the server of the highestPlUserId isn't possibly in there twice?